applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -1279,6 +1467,93 @@ static int cmd_program_info(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+#define FILE_LIST_BUF_SIZE 512
+static int cmd_file_list(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	char buf[FILE_LIST_BUF_SIZE];
+	size_t buf_size = FILE_LIST_BUF_SIZE;
+	char *dir_path = NULL;
+
+	if (argc > 2) {
+		shell_error(shell, "Usage: bct file list [dir path]");
+		return -EINVAL;
+	}
+
+	if (argc == 2) {
+		dir_path = argv[1];
+	}
+
+	ret = sd_card_list_files(dir_path, buf, &buf_size);
+	if (ret) {
+		shell_error(shell, "List files err: %d", ret);
+		return ret;
+	}
+
+	shell_print(shell, "%s", buf);
+
+	return 0;
+}
+
+static int cmd_file_select(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	uint8_t big_idx;
+	uint8_t sub_idx;
+	uint8_t bis_idx;
+
+	if (argc < 4) {
+		shell_error(shell,
+			    "Usage: bct file select <file path> <BIG index> <subgroup index> "
+			    "<BIS index> [loop file 1/0]");
+		return -EINVAL;
+	}
+
+	ret = argv_to_indexes(shell, argc, argv, &big_idx, 2, &sub_idx, 3);
+
+	if (!is_number(argv[4])) {
+		shell_error(shell, "BIS index must be a digit");
+		return -EINVAL;
+	}
+
+	bis_idx = (uint8_t)atoi(argv[4]);
+
+	if (bis_idx >= CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT) {
+		shell_error(shell, "BIS index out of range");
+		return -EINVAL;
+	}
+
+	char *file_name = argv[1];
+	bool loop_file = false;
+
+	if (argc == 6) {
+		bool valid_argument =
+			is_number(argv[5]) && ((atoi(argv[5]) == 1) || (atoi(argv[5]) == 0));
+
+		if (!valid_argument) {
+			shell_error(shell, "Loop file must be 0 or 1");
+			return -EINVAL;
+		}
+
+		loop_file = (atoi(argv[5]) == 1);
+	}
+
+	LOG_INF("Selecting file %s for stream %d.%d.%d", file_name, big_idx, sub_idx, bis_idx);
```

Comment:
Nit. Align with Alexander on format, but suggest
```suggestion
	LOG_INF("Selecting file %s for stream bis: %d sub: %d bis: %d", file_name, big_idx, sub_idx, bis_idx);
```

Reply:
I have no strong opinion - @alexsven comment if you have another format in mind

Reply:
Probably don't need this at all, but could be a DBG. I want people to use the `bct show` to check if their configs are correct before the start broadcasting.